src/hotspot/share/gc/shenandoah/shenandoahFullGC.cpp

```diff
@@ -176,9 +176,14 @@ void ShenandoahFullGC::op_full(GCCause::Cause cause) {
   if (heap->mode()->is_generational()) {
     size_t old_available = heap->old_generation()->available();
     size_t young_available = heap->young_generation()->available();
-    log_info(gc, ergo)("At end of Full GC, old_available: " SIZE_FORMAT "%s, young_available: " SIZE_FORMAT "%s",
+    log_info(gc, ergo)("At end of Full GC,  old_available: " SIZE_FORMAT "%s out of total: " SIZE_FORMAT "%s,"
```

Comment:
Consider `ShenandoahGeneration::log_status`.

Reply:
I've integrated the two heap-status logging approaches.  Let me know what you think...  Thanks for suggestion.